weekly/2016/1012.md

```diff
@@ -16,4 +16,72 @@
 [qsy]: https://github.com/JasonQSY
 [tq]: https://github.com/tq5124
 
+## Content
 
+- [Service](#user-content-service)
+  - [Ryver](https://ryver.com/)
+- [Lists](#user-content-lists)
+  - [Open source at Instacart](https://www.instacart.com/opensource)
+  - [Mega Project List](https://github.com/karan/Projects)
+  - [Static Site Generators](https://staticsitegenerators.net/)
+- [Great Works](#user-content-great-works)
+  - [MyScript Developer](https://dev.myscript.com/) with a [demo](https://webdemo.myscript.com/views/math.html)
```

Comment:
I think it's better to mention this demo can recognize hand writing and give latex code